### ISSUE TITLE
Is this hostable?

### ISSUE THREAD

**Issue by will-hill** [louisnw01/lightweight-charts-python#329]
### Question

Is it possible to host this app on a server like **Firebase Functions, Cloud Run** or even **Vercel** using **Flask** or **Fast API**?

I was able to build a chart with indicators and backtesting extremely fast.  To do this in React or JS would take my days due to my lack of web coding skills.  Searching around I see there might be a way to do it using Flask but wanted to verify before I dug in too deeply.

PS This app is awesome; the quality of the API and docs is amazing.  I hope my sponsorship helps.

![Screenshot 2024-04-02 at 9 33 32 AM](https://github.com/louisnw01/lightweight-charts-python/assets/8185572/50f7cb59-d49b-4bdd-aab0-26c7a6f017e0)

**Comment by louisnw01** [comment]
Hey @will-hill,

It certainly does help, thank you very much!

The library relies on evaluating JavaScript within the DOM, so it wouldn't be possible to host the app.

In order to achieve this functionality you'd need to pass all communication through websockets, which unfortunately would require a rewrite of the entire library.

Louis

**Comment by mehboob-dev** [comment]
Hey @will-hill,

Can you please guide me how to bring the trades in legends, with that eye icon ?
Thanks!

**Comment by cilindro5432** [comment]
I started creating a lightweight charts python web app with the previous version in Django and it worked very well, I would say very similar to the desktop app. I only had problems recognizing the 2 clicks to create the ray and trend lines but horizontal worked fine. I used abstract.py as a base but when using it in Django I had to use other code that could manipulate the JS files. Never try it with websocket or live data, only static data, I don't know if it works like in the desktop app. But definitely much faster. I like this project it feels better than working with Trading View.
